optaplanner-operator/pom.xml

```diff
@@ -24,19 +24,12 @@
   <properties>
     <java.module.name>org.optaplanner.operator</java.module.name>
     <!-- Quarkus Ecosystem CI (and other CI possibly too) updates the version.io.quarkus property. -->
-    <version.io.quarkus.override>2.13.0.Final</version.io.quarkus.override>
+    <version.io.quarkus.override>2.15.0.Final</version.io.quarkus.override>
```

Comment:
Of course, we need to keep the override as if we downgrade the Quarkus version to the last LTS, the operator-sdk breaks on Jandex.